tests/cpputest/unittests/testsSerialization.cpp

```diff
@@ -167,12 +169,12 @@ TEST(returnDataSerialization, testString) {
     int nk = 2;
     int nx = 3;
     int nz = 4;
-    amici::UserData u(np, nx, nk);
     amici::Model_Test m( nx, nx, 4, 4, nz, nz, 8, 9, 10, 11, 12, 13, 14, 15, amici::AMICI_O2MODE_NONE,
                    std::vector<realtype>(np,0.0),std::vector<realtype>(nk,0.0),std::vector<int>(np,0),
                    std::vector<realtype>(nx,0.0),std::vector<int>(nz,0));
 
-    amici::ReturnData r(&u, &m);
+    amici::CVodeSolver solver;
+    amici::ReturnData r(solver, &m);
 
     std::string serialized = amici::serializeToString(r);
 
```

Comment:
remove changes in writeResults.h5 ?, probably good to have a script that automatically resets that file

Reply:
Why did we add this one to the repository again?

Reply:
hmm, might be that I did not figure out how to create hdf5 files from c++.